README.md

```diff
@@ -1 +1,97 @@
-# Hello-world
\ No newline at end of file
+# HTML & CSS3 Course
+
+If you are not familiar with linters, read [root level README](../README.md).
```

Comment:
Kindly add a description for your PR with what you did exactly in this PR in order for your PR to be professional.